es_managers/es_idx_manager/build_deploy_config.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
I wonder if we should put one copy of this somewhere else file in the repo and then use it for both. I would probably only require a couple changes here for this to work.